src/components/course-settings/remove-teacher.cjsx

```diff
@@ -25,10 +24,11 @@ module.exports = React.createClass
       .value()
 
   goToDashboard: ->
-    @context.router.transitionTo('dashboard', {courseId: @props.courseId})
+    TeacherRosterStore.once 'deleted', ->
+      window.location.href = '/dashboard/'
```

Comment:
Oh also, should the delete button in the modal be an `AsyncButton` so the user sees if the delete failed?